app/data/models.py

```diff
@@ -34,7 +35,45 @@ class ActionTypes(object):
             (UPDATE, 'Update'),
             (DELETE, 'Delete')
         )
+
         @classmethod
         def as_list(cls):
             return [cls.CREATE, cls.UPDATE, cls.DELETE]
+
     action = models.CharField(max_length=6, choices=ActionTypes.choices)
+
+
+class CeleryJob(models.Model):
+    """ Stores information about a celery job
+    """
+
+    class Status(object):
+        """Status of job"""
+        PENDING = 'PENDING'
+        STARTED = 'STARTED'
+        SUCCESS = 'SUCCESS'
+        ERROR = 'ERROR'
+        CHOICES = (
+            (PENDING, 'Pending'),
+            (STARTED, 'Started'),
+            (SUCCESS, 'Success'),
+            (ERROR, 'Error'),
+        )
+
+    class TaskType(object):
+        DEDUPE = 'DEDUPE'
+
+    uuid = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    date = models.DateTimeField(auto_now_add=True, db_index=True)
```

Comment:
This is just like the one above, but my preference would be for a DateTime field to be called 'datetime' (or 'created') rather than just 'date'.